packages/core/src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -201,8 +273,16 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
           tooltipProps={tooltipProps}
           weight={weight}
           type={type}
+          ellipsis={!multiline}
         >
-          {inputValue || placeholder}
+          {(multiline
+            ? inputValue.split("\n").map((line, index) => (
+                <React.Fragment key={index}>
+                  {line}
+                  {index < inputValue.split("\n").length - 1 && <br />}
+                </React.Fragment>
+              ))
```

Comment:
Why do we need this? Textareas know how to break lines

Reply:
this is not a textarea, that is the TypographyComponent (Text) that is shown once the textarea is not in edit mode any more

Reply:
why not just set ellipsis to false?

Reply:
ellipsis is for wrapping
we're trying to have "new lines" supported
a div needs <br/> to break into a new line

Reply:
by the way it is with ellipsis false

Reply:
You can use only `white-space: pre-wrap` and it will behave the same as in editing mode and avoid splitting the string with `<br />`s

Reply:
@talkor works like a charm
changed it